Updates for two mc releases (#1160)
minio/docs: source/reference/minio-mc/mc-get.rst

==========
``mc get``
==========

.. default-domain:: minio

.. contents:: Table of Contents
   :local:
   :depth: 2

.. mc:: mc get

.. versionadded:: mc RELEASE.2024-02-24T01-33-20Z

Syntax
------

.. start-mc-get-desc

> The :mc:`mc get` downloads an object from a target S3 deployment to the local file system.

Reviewer: probably
```suggestion
The :mc:`mc get` command downloads an object from a target S3 deployment to the local file system.
```

.. end-mc-get-desc

> Unlike other commands that can download files, such as :mc:`mc cp` or :mc:`mc mirror`, ``mc get`` is a simpler call.

Reviewer 2: ```suggestion
``mc get`` provides a simplified interface for downloading files compared to :mc:`mc cp` or :mc:`mc mirror`.

``mc get`` uses a one-way download function that trades efficiency for the power and complexity of the other commands.
```

Otherwise we might be implying that `mc cp` and `mc mirror` are 'bad' in some way.

> Unlike other commands that can download files, such as :mc:`mc cp` or :mc:`mc mirror`, ``mc get`` is a simpler call.
> ``mc get`` only does the one-way function of downloading the file so that it avoids the potential performance costs of other commands.

Reviewer 2: ```suggestion
``mc get`` provides a simplified interface for downloading files compared to :mc:`mc cp` or :mc:`mc mirror`.

``mc get`` uses a one-way download function that trades efficiency for the power and complexity of the other commands.
```

Otherwise we might be implying that `mc cp` and `mc mirror` are 'bad' in some way.

Author: Good point.

.. tab-set::

   .. tab-item:: EXAMPLE

      The following downloads the file ``logo.png`` from an s3 source to the local file system at path ``~/images/collateral/``.

      .. code-block:: shell
         :class: copyable

         mc get minio/marketing/logo.png ~/images/collateral

   .. tab-item:: SYNTAX

      The command has the following syntax:

      .. code-block:: shell
         :class: copyable

         mc [GLOBALFLAGS] get                      \
                          SOURCE                   \
                          TARGET                   \
                          [--encrypt-key value]    \
                          [--encrypt value] 

      .. include:: /includes/common-minio-mc.rst
         :start-after: start-minio-syntax
         :end-before: end-minio-syntax

Parameters
~~~~~~~~~~

.. mc-cmd:: SOURCE
   :required:

   The full path to the :ref:`alias <minio-mc-alias>`, bucket, prefix (if used), and object of the path to download.

.. mc-cmd:: TARGET
   :required:

>    The path on the local file system where the object should download to.

Reviewer: An idea, perhaps not a great one, but maybe some variation on this:
```suggestion
   The destination path for the downloaded object on the local filesystem.
```

Author: I did some things to remove the ending preposition.
